nx-libs.spec

```diff
@@ -38,7 +38,11 @@ BuildRequires:  pkgconfig(pixman-1) >= 0.13.2
 BuildRequires:  pkgconfig(x11)
 BuildRequires:  pkgconfig(xext)
 BuildRequires:  pkgconfig(xpm)
+%if 0%{?suse_version} >= 4210
```

Comment:
This is what originally mislead me as well. It's not part of the official OpenSUSE repository though, but a custom one. Like our repositories for Debian...